docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,147 @@
+# Tutorial: Deploy the Chrony charm
+
+## What you'll do
+This tutorial will walk you through deploying the chrony charm; you will:
+1. Deploy and configure the Chrony charm
+2. Enable NTS for the Chrony charm
+3. Use the Chrony charm as a time server
+
+## Prerequisites
+* LXD installed and bootstrapped.
+* A host machine with Juju version 3.4 or above and a Juju controller bootstrapped.
+
+## Create a Juju model
+Since the clock is a kernel function, we will need to use LXD virtual machines
+instead of containers.
+```
+juju add-model test-chrony
+juju set-model-constraints virt-type=virtual-machine
+```
+
+## Deploy the chrony charm and confirm the chrony charm are deployed
+1. Deploy the latest chrony charm.
+```
+juju deploy chrony --channel latest/edge
+```
+2. Run `juju status`. You should see something like this (IPs will be different):
+```
+Model          Controller  Cloud/Region      Version  SLA          Timestamp
+test-chrony    lxd         lxd/default       3.5.2    unsupported  07:20:46Z
+
+App     Version  Status   Scale  Charm   Channel      Rev  Exposed  Message
+chrony           blocked      1  chrony  latest/edge   35  no       no time source configured
+
+Unit       Workload  Agent  Machine  Public address  Ports    Message
+chrony/0*  blocked   idle   0        10.81.171.230   123/udp  no time source configured
+
+Machine  State    Address        Inst id        Base          AZ  Message
+0        started  10.81.171.230  juju-728b65-0  ubuntu@24.04      Running
+```
+
+## Configure the time source for chrony charm
+Since Chrony needs a time source before it can serve time to others, 
+we must configure a time source for the Chrony charm. 
```

Comment:
Nitpick: For conciseness, I would rephrase this sentence as "We must configure a time source for the Chrony charm so it can serve time to others."

For my understanding, the Chrony charm serves time to other...charms? Might be useful to clarify if it's a specific type of "others"

Reply:
Yes, indeed this is better, and chrony charm serves time to all other NTP clients, so how about: "We must configure a time source for the Chrony charm so it can serve time other NTP clients."?

Reply:
Thank you for the clarification! Your suggestion looks good but might be missing a "to"? More explicitly:

> "We must configure a time source for the Chrony charm so it can serve time **to** other NTP clients."

Reply:
Updated, thanks!